Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'redis-client', '>= 0.11.2'
 gem 'redlock'
 gem 'sidekiq', '< 8'
 gem 'sidekiq-scheduler'
+gem 'whenever', require: false
```

Comment:
I'm not convinced by us adding a new way of scheduling jobs currently. Why would we use this rather than sidekiq-scheduler?

Reply:
ahhh i didnt see this! yeah happy to use that instead 👍

Reply:
Even though realise it was literally the line above 😂

Reply:
No worries :rocket: :smile: